lib/engines/bh.js

```diff
@@ -27,9 +28,9 @@ module.exports = function (opts) {
 
         filePath = U.fulfillName(name, opts.ext || this.ext, opts.source);
 
-        // check cache options
-        if (opts.force || options.forceExec || options.forceLoad) {
```

Comment:
@zxqfox you wrote that these options are important. Can you explain to?
If we support them, we should describe in documentation how to use them.